libs/npm/core/src/components/grid-table/Pager.tsx

```diff
@@ -65,6 +66,29 @@ export const Pager: React.FC<IPagerProps> = ({
   setPageSize,
   manualPageSize,
 }) => {
+  const { search } = useLocation();
+  let query = React.useMemo(() => new URLSearchParams(search), [search]);
+  let [searchParams, setSearchParams] = useSearchParams(query);
+  const qtyCookie = document.cookie
+    .split('; ')
+    .find((row) => row.startsWith('qty='))
+    ?.split('=')[1];
+
+  // update cookie when qty changes
+  React.useEffect(() => {
+    if (qtyCookie !== searchParams.get('qty') && searchParams.get('qty') !== null)
+      document.cookie = `qty=${searchParams.get('qty')}; expires=Fri, 31 Dec 9999 23:59:59 GMT`;
```

Comment:
Do you need to hardcode the date?

Reply:
For cookies we can either set a time limit or a far future date. By default it ends after the user closes their browser unfortunately